hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -350,6 +350,10 @@ public long getExcludeNodesExpiryTime() {
     return excludeNodesExpiryTime;
   }
 
+  public void setExcludeNodesExpiryTime(long expiryTime) {
```

Comment:
unable to find its caller in changed code, please check if this is required

Reply:
oh you're right, this is not required. Thanks!